realms/lib/util.py

```diff
@@ -71,10 +71,31 @@ def extract_name(file_path):
     return os.path.basename(file_path)
 
 
+def get_ext(path):
+    match = re.match(r"^[^.]+\.([^.]+)$", path)
+    if match:
+        return match.group(1)
+    else:
+        return ''
+
+
 def remove_ext(path):
     return re.sub(r"\..*$", "", path)
 
 
```

Comment:
Why don't you use `os.path.splitext` here?

Reply:
That's a good point, I didn't know about that particular function and will change the code to use that.